benches/less_than.rs

```diff
@@ -1,5 +1,7 @@
 #![allow(unused)]
 
+mod group;
+
```

Comment:
This seems unnecessary?

Reply:
Right, fixed in https://github.com/privacy-scaling-explorations/halo2curves/pull/26/commits/94202846e0610c6ae74415949173f1200e3dec2b